README.md

```diff
@@ -15,12 +31,92 @@ A set of tools to simplify working with gRPC
 ```
 
 ## swagger-protoc-gen
 
-```
+Generates swagger json from proto files via `protoc-gen-swagger` from [grpc-gateway](https://github.com/grpc-ecosystem/grpc-gateway) providing default include path. Code generation can be started by command
+
+```shell
 $ swagger-protoc-gen --swagger_opt=json_names_for_fields=true --swagger_out=logtostderr=true:./tests/generated/server/ -I ./tests/protos/ greeter.proto
 ```
+
+# Contributing
+
+To contribute, you will need to setup your local environment, see [prerequisites](#prerequisites). For the contribution and workflow guide, see [package development lifecycle](#package-development-lifecycle).
+
+A detailed overview on how to contribute can be found in the [contributing guide](CONTRIBUTING.md).
+
+## Prerequisites
+
+Make sure you have installed all of the following prerequisites on your development machine:
+
+- Git - [Download & Install Git](https://git-scm.com/downloads). OSX and Linux machines typically have this already installed.
+- Node.js (version 10 or higher) - [Download & Install Node.js](https://nodejs.org/en/download/) and the npm package manager.
+
+## Code conventions
+
+For Node.js code formatting is suggested to use [prettier](https://github.com/prettier/prettier) formatter ([Visual Studio Code extension for Prettier](https://github.com/prettier/prettier-vscode)). Shared prettier settings are saved at [.prettierrc file](/.prettierrc)
+
+## General folders layout
+
+### .vscode
+
+- configs for Visual Studio Code.
+
+### build
+
+- scripts for the CI build process
+
+### deps
+
+- repository submodules
+
+### src
+
+- `src/bin` - tools entry points.
+- `src/include` - well known proto files, copied from other repositories, which are generated by command `npm run build:copy`. Folder is created automatically during the build process and was excluded from repository.
+- `*` - implementation of the service.
+
+### tests
+
+- tests for the result of code generation.
+
+## Package development lifecycle
+
+- Declare new tool in the `bin` section `package.json`
+- Implement tool entry point in `src/bin`
+- Implement tool logic in `src`
+- Add or addapt unit-tests (prefer before and simultaneously with coding)
+- Add or change the documentation as needed
+- Open pull request in the correct branch. Target the project's `develop` branch
+
+## Unit testing
+
+### Files layout
+
+- `src/protos` - files on which code generation can be tested.
+- `tests/generated` — artifacts of the proto files compilation, which are generated by command `npm run build`. Folder is created automatically during the build process and was excluded from repository.
+- `tests/\*.test.js` — unit test cases
+
+### Running tests
+
+Tests can be run using the following commands.
+
+```
+$ npm i
+$ npm run build
+$ npm t
+```
+
+# Maintainers
+
+@Byndyusoft/owners: https://github.com/orgs/Byndyusoft/teams/owners
+
+Team maintainers:
```

Comment:
maybe it's better to use corporate emails?

Reply:
or create email group like github.maintain@byndyusoft.com

Reply:
done